src/display/annotation_layer.js

```diff
@@ -1065,7 +1065,7 @@ class TextWidgetAnnotationElement extends WidgetAnnotationElement {
       const elementData = {
         userValue: textContent,
         formattedValue: null,
-        valueOnFocus: "",
+        lastCommittedValue: "",
```

Comment:
Given that you reset the value to `null` in one case below (i.e. "beforeinput"), should the initial value perhaps also be `null` here?